src/rocksdb_options.rs

```diff
@@ -262,6 +262,30 @@ impl WriteOptions {
     }
 }
 
+pub struct CompactOptions {
+    pub inner: *mut DBCompactOptions,
+}
+
+impl CompactOptions {
+    pub fn new() -> CompactOptions {
+        unsafe { CompactOptions { inner: crocksdb_ffi::crocksdb_compactoptions_create() } }
+    }
+
+    pub fn set_exclusive_manual_compaction(&mut self, v: bool) {
```

Comment:
It is means when manual compaction is running, other background compaction threads will stop. We need set it false in our case.